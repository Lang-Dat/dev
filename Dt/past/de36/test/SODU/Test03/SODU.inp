5 16 375
4 13
5 8
2 8
7 12
2 11
3 10
6 12
1 11
5 8
6 10
7 10
6 8
3 10
2 12
5 12
3 14
5 10
7 13
2 12
5 15
1 10
1 12
6 10
4 13
4 14
3 12
3 13
3 8
7 10
5 11
5 11
6 11
1 15
3 10
2 12
5 8
5 8
5 13
7 11
7 10
5 13
3 10
6 13
7 13
3 9
6 14
6 10
4 9
2 15
3 15
5 11
3 13
4 9
1 13
5 8
4 9
2 15
4 10
3 10
2 11
6 9
5 14
7 10
2 8
4 15
4 12
5 8
2 9
6 14
5 13
6 11
7 9
6 15
7 11
5 10
3 13
4 15
2 12
2 9
2 8
2 12
7 10
1 8
5 14
1 12
1 14
6 12
6 10
6 14
1 15
1 9
3 8
7 9
4 15
1 8
1 10
5 13
5 10
5 8
6 8
1 14
4 10
2 12
5 15
4 14
6 8
2 9
2 13
5 10
6 15
2 13
4 9
7 13
1 9
5 8
2 9
6 13
2 15
4 14
1 8
4 9
5 10
1 8
3 13
1 11
2 9
7 15
1 14
6 10
3 9
3 10
6 15
3 11
4 13
1 11
6 8
6 8
5 13
3 11
1 9
6 12
5 11
3 8
1 9
5 9
7 15
3 10
7 9
4 15
7 12
5 11
2 13
4 8
3 8
7 11
5 12
4 13
6 11
6 10
2 15
2 11
4 8
1 9
6 8
4 12
7 9
7 13
4 13
3 8
7 15
6 9
4 13
3 13
7 13
5 9
3 14
7 10
1 15
2 14
4 14
1 15
7 9
1 11
1 8
2 10
5 12
4 14
7 8
2 9
1 12
7 14
3 14
3 12
2 10
2 14
1 8
7 9
7 14
3 13
7 8
3 13
1 15
3 13
3 10
1 9
3 15
5 10
6 15
1 14
7 14
4 14
4 12
2 13
3 8
1 10
4 12
5 10
5 11
1 12
6 13
4 10
1 8
6 9
7 13
7 15
4 10
6 11
7 13
6 11
7 13
6 11
3 9
7 10
5 14
1 9
1 15
6 10
2 12
3 15
2 11
5 12
3 13
1 10
5 10
4 8
2 11
4 8
2 13
5 15
1 9
3 14
6 8
3 11
1 10
5 8
4 12
4 13
3 9
3 10
1 13
6 14
7 11
2 10
1 15
4 10
2 9
7 11
5 14
7 12
1 10
5 11
3 10
1 8
3 15
4 8
4 12
1 14
4 15
3 14
6 15
4 9
3 10
1 15
7 15
7 11
3 11
1 10
2 10
1 10
4 14
1 13
2 15
7 13
7 13
3 14
7 15
7 13
5 10
3 14
5 13
4 9
7 14
4 10
1 10
6 14
3 11
1 8
5 13
6 14
2 9
4 13
5 10
3 11
4 15
6 13
1 13
5 15
1 12
3 13
4 14
1 15
6 11
4 11
3 13
2 13
3 13
6 8
6 13
3 14
5 14
4 15
5 9
1 9
7 15
5 13
7 10
2 13
2 13
7 12
3 11
6 11
1 11
5 10
2 13
5 13
3 9
7 13
3 12
7 13
6 9
1 12
5 14
6 10
3 9
1 11
4 8
1 8
3 15
6 13
4 8
3 11
3 15
4 13
5 11
4 15
7 11
7 8
3 13
4 9
6 13
4 11
7 13
5 14
4 14
4 10

