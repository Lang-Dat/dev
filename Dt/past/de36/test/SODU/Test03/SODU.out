14
12
3
10
14
8
3
3
12
5
12
7
8
15
8
11
10
15
15
13
6
4
5
14
14
6
11
10
12
7
7
2
9
8
15
12
12
13
9
12
13
8
8
15
15
8
5
2
4
11
7
11
2
9
12
2
4
11
8
14
12
13
12
3
14
9
12
8
8
13
2
3
8
9
10
11
14
15
8
3
15
12
8
13
4
9
3
5
8
9
13
10
3
14
8
6
13
10
12
7
9
11
15
13
14
7
8
4
10
8
4
2
15
13
12
8
8
4
14
8
2
10
8
11
3
8
15
9
5
15
8
8
5
14
3
7
7
13
5
13
3
7
10
13
1
15
8
3
14
10
7
4
13
10
9
8
14
2
5
4
14
13
13
7
9
3
15
14
10
15
12
14
11
15
1
11
12
9
4
14
9
3
3
8
1
8
14
14
8
4
15
11
6
1
4
8
3
15
11
14
11
9
11
8
13
11
10
8
9
15
14
9
4
10
6
9
10
7
4
8
11
8
12
15
15
11
2
15
2
15
2
15
12
13
13
9
5
15
11
14
8
11
6
10
13
14
13
4
13
13
11
7
5
6
12
9
14
15
8
9
8
9
1
9
11
8
9
13
10
6
7
8
8
11
13
9
9
14
11
8
2
8
9
15
9
5
6
1
6
14
9
4
15
15
11
15
15
10
11
13
2
15
11
6
8
5
8
13
8
8
14
10
5
14
8
9
13
4
11
14
9
2
8
11
4
11
7
8
11
13
14
1
13
15
13
12
4
4
10
5
2
3
10
4
13
15
15
6
15
12
4
13
5
15
3
13
8
11
8
13
5
11
14
7
14
9
14
11
2
8
8
15
13
14
11

